pandas/core/generic.py

```diff
@@ -1884,6 +1884,10 @@ def keys(self) -> Index:
         Index
             Info axis.
 
+        See Also
+        --------
+        Series.keys : Return alias for index.
```

Comment:
This is referencing the existing method. I would suggest `.index` and `.columns`